.gitmodules

```diff
@@ -10,3 +10,7 @@
 [submodule "utils/kokkos"]
 	path = utils/kokkos
 	url = https://github.com/kokkos/kokkos.git
+[submodule "utils/pybind11"]
+	path = utils/pybind11
+	url = https://github.com/pybind/pybind11
+	branch = stable
```

Comment:
Is PyBind as a submodule the preferred way to do this? Or should we use, e.g., fetch content through cmake?

Reply:
It was just the easiest to do for a quick prototype. I can definitely change it to use FetchContent instead.

Reply:
I think that's desirable, at least for now. We've been trying to move away from submodules when possible.